CHANGELOG.md

```diff
@@ -1,5 +1,286 @@
 # Changelog
 
+## v0.7.0
+
+### Breaking changes for users
+
+* new base class names for pages, components and apps, issue #36 (@jonasjabari)
+* simplified custom component structure/class names #39 (@jonasjabari)
+* improved vue.js component naming convention #41 (@jonasjabari)
+* `pg` component is now called `paragraph`, issue #47, pull #48 (@PasWen)
+
+### Improvements for users
+
+* added `hr` component, pull #49, (@michaelrevans)
+* allow actions to accept id and class attributes, issue #44, pull #50 (@michaelrevans)
+
+
+### Breaking changes for core developers
+
+* namespaced core components using `Matestack::Ui::Core` module/folder
+* simplified core components folder structure, (aligned with issue #39), pull #64 (@jonasjabari)
+* changed vue.js component naming
+* add-on engine components now need `Matestack::Ui` namespacing
+
+### Improvements for core developers
+
+* started to move business logic out of `app/concepts` into `app/lib` folder in order to create a better structure (in progress --> core refactoring)
+
+### Migration guide for users from v0.6.0
+
+#### App base class name
+
+***OLD:***
+
+`app/matestack/app/example_app.rb`
+```ruby
+class Apps::ExampleApp < App::Cell::App end
+```
+
+***NEW:***
+
+`app/matestack/app/example_app.rb`
+```ruby
+class Apps::ExampleApp < Matestack::Ui::App end
+```
+
+#### Page base class name
+
+***OLD:***
+
+`app/matestack/pages/example_app/example_page.rb`
+```ruby
+class Pages::ExampleApp::ExamplePage < Page::Cell::Page end
+```
+
+***NEW:***
+
+`app/matestack/pages/example_app/example_page.rb`
+```ruby
+class Pages::ExampleApp::ExamplePage < Matestack::Ui::Page end
+```
+
+#### Custom STATIC component base class name / folder structure
+
+***OLD:***
+
+`app/matestack/components/card/cell/card.rb`
+```ruby
+class Components::Card::Cell::Card < Component::Cell::Static end
+```
+
+***NEW:***
+
+`app/matestack/components/card.rb`
```

Comment:
No additional `/card/` folder here?

Reply:
no, it should be super simple to add custom components without an extra order and module name